src/main/java/com/odysseusinc/arachne/datanode/service/impl/UserServiceImpl.java

```diff
@@ -79,27 +80,19 @@
     @Autowired
     private DataNodeService dataNodeService;
     @Autowired
-    private CommonUserDTOToUserConverter commonUserDTOToUserConverter;
+    private ApplicationEventPublisher eventPublisher;
 
-    @PostConstruct
-    public void syncUsers() {
+    @Override
+    public User create(User user) {
 
-        try {
+        user.setEnabled(true);
+        user.getRoles().add(getAdminRole());
+        if (Objects.equals(FunctionalMode.NETWORK, dataNodeService.getDataNodeMode())) {
             dataNodeService.findCurrentDataNode().ifPresent(dataNode -> {
-                        LOG.info(RELINKING_ALL_USERS_LOG);
-                        final List<User> users = userRepository.findAll();
-                        List<User> updatedUsers = centralIntegrationService.relinkAllUsersToDataNodeOnCentral(dataNode, users);
-                        List<User> mappedUsers = users.stream().map(user -> {
-                            updatedUsers.stream().filter(u -> u.getEmail().equals(user.getEmail())).findFirst()
-                                    .ifPresent(u -> user.setEnabled(u.getEnabled()));
-                            return user;
-                        }).collect(Collectors.toList());
-                        userRepository.save(mappedUsers);
-                    }
-            );
-        } catch (Exception ex) {
-            LOG.error(RELINKING_ALL_USERS_ERROR_LOG, ex.getMessage());
+                centralIntegrationService.linkUserToDataNodeOnCentral(dataNode, user);
```

Comment:
Sync flag should be set here otherwise the Node will fail to restart in Network mode